packages/opentelemetry-node/lib/instrumentations.js

```diff
@@ -259,6 +259,15 @@ function getInstrumentations(opts = {}) {
         'OTEL_NODE_DISABLED_INSTRUMENTATIONS'
     );
 
+    // `@opentelemetry/instrumentation-http` defaults to emit old semconv attributes.
+    // Set the default to stable HTTP semconv if not defined by the user (http, http/dup)
```

Comment:
Could add a note to the comment that we could drop this one we dep on a release of instr-http with https://github.com/open-telemetry/opentelemetry-js/pull/5552